File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        left, right, current = 0,len(nums)-1,0

        while current <= right:
            if nums[current] == 0:
                t = nums[current]
                nums[current] = nums[left]
                nums[left] = t
                left += 1
                current += 1 
                continue
            elif nums[current] == 2:
                t = nums[current]
                nums[current] = nums[right]
                nums[right] = t
                right -= 1
            else:
                current += 1
        
        

File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int current = nums[0];
        int best = nums[0];
        for(int i=1; i<nums.length; i++){
            current = Math.max(nums[i], current+nums[i]);
            best = Math.max(best, current);
        }
        return best;
    }
}
